integration-tests/camel-aws-s3/src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+quarkus.ssl.native=true
+
+camel.context.name=quarkus-camel-aws-s3
+camel.properties.prefixToken={{
+camel.properties.suffixToken=}}
+camel.conf={{env:CAMEL_CONF:}}
+camel.confd={{env:CAMEL_CONFD:}}
+camel.routes.dump=true
+camel.routes.defer=true
+camel.routes.locations=
+
+camel.component.aws-s3.access-key={{env:S3_ACCESS_KEY}}
+camel.component.aws-s3.secret-key={{env:S3_SECRET_KEY}}
+camel.component.aws-s3.region={{env:S3_REGION}}
```

Comment:
Hmmm, this needs to be discussed with @stuartwdouglas as we would need to have this set up on CI somehow. Did you already check with him?

Reply:
Yes, I tested everything.

Reply:
Btw actually the integration tests related to camel are commented out.

Reply:
I commented out the Integration tests until we found a way to use an AWS account on CI